media-sound/whipper/whipper-0.7.0-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
For the record, I'm not 100 % sure about the policy about `git mv`, you might get asked to split -r1 bump & removal of -r0 to two separate commits.